docs/README.md

```diff
@@ -9,6 +9,7 @@ subdirectories:
 - [Russian translation guide / Участие в переводе](ru/translation-guide.md)
 - [Simplified Chinese Guide / 简体中文翻译指南](zh-cn/translation-guide.md)
 - [Traditional Chinese Guide / 正體中文翻譯指南](zh-tw/translation-guide.md)
+- [Spanish guide / Guía en español](es/README.md)
```

Comment:
I like the idea of using `README.md` but seeing other teams it seems like the standard is `translation-guide.md` and we may be sticky to that convention